client/lib/utils/phone-format.js

```diff
@@ -0,0 +1,27 @@
+import { PhoneNumberUtil, AsYouTypeFormatter } from 'google-libphonenumber';
+
+export const isValidPhone = ( phoneNumber, country ) => {
+	const phoneUtil = PhoneNumberUtil.getInstance();
```

Comment:
No need for that. I've checked the source and it looks like it's using the "typical" Java Singleton pattern: https://github.com/google/closure-library/blob/969c7e53decf7b6ee3b2dd14011663d0762a1e82/closure/goog/base.js#L755

So after the first time `getInstance()` is called, the subsequent calls are fast, the `PhoneNumberUtil` object isn't being re-initialized again and again.